calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+| `max_calibration_range`                     | `double`      | `50.0`                                                  | Maximum range for calibration in meters.                                                                                                                           |
+| `background_model_timeout`                  | `double`      | `5.0`                                                   | The background model will terminate if there are no new points in the background within this period, measured in seconds.                                          |
+| `min_foreground_distance`                   | `double`      | `0.4`                                                   | Minimum distance in meters for extracting foreground points.                                                                                                       |
+| `background_extraction_timeout`             | `double`      | `15.0`                                                  | Timeout in seconds for background extraction processes.                                                                                                            |
+| `ransac_threshold`                          | `double`      | `0.2`                                                   | Distance threshold for the segmentation model.                                                                                                                     |
+| `ransac_max_iterations`                     | `int`         | `100`                                                   | Maximum number of iterations for the segmentation model.                                                                                                           |
+| `lidar_cluster_max_tolerance`               | `double`      | `0.5`                                                   | Maximum cluster tolerance for extracting lidar cluster.                                                                                                            |
+| `lidar_cluster_min_points`                  | `int`         | `3`                                                     | Minimum number of points required to form a valid lidar cluster.                                                                                                   |
+| `lidar_cluster_max_points`                  | `int`         | `2000`                                                  | Maximum number of points allowed in a lidar cluster.                                                                                                               |
+| `radar_cluster_max_tolerance`               | `double`      | `0.5`                                                   | Maximum cluster tolerance for extracting radar cluster.                                                                                                            |
```

Comment:
units

Reply:
fixed in https://github.com/tier4/CalibrationTools/commit/69cf81c260d5f15c08f51f808c14ac2d8d79b30d